src/foam/u2/view/RichChoiceView.js

```diff
@@ -31,9 +31,11 @@ foam.CLASS({
                 dao: X.userDAO.orderBy(foam.nanos.auth.User.LEGAL_NAME)
               },
               // Set "disabled: true" to render each object as non-selectable row
+              // Set "hideOnEmpty: true" to hide headers if not objects are present in dao provided.
               {
                 disabled: true,
                 heading: 'Disabled users',
+                hideOnEmpty: true,
```

Comment:
"On" in a name makes me think it's an event listener, like onClick. A section being empty isn't an event (imo) so I think a better name would be `hideIfEmpty`.

Reply:
I was juggling between the two but I'll take that as a vote for if instead of on. :+1: